mk/gx.mk

```diff
@@ -2,6 +2,7 @@ gx-path = gx/ipfs/$(shell gx deps find $(1))/$(1)
 
 gx-deps:
 	gx install --global
+	gx-go rw
```

Comment:
Rewriting as part of the build process will make my workflow of working with a partly rewritten tree impossible or at least difficult.

Reply:
A quick fix for this might be to provide an official way to build without automatically calling the `gx-deps` target...

Reply:
@kevina Could describe how you work with a partly rewritten tree? I can't think of a good use case for this, I am not saying there isn't a good use case for it, I just can't think of it :slightly_smiling_face: